regulations/settings/base.py

```diff
@@ -173,6 +173,15 @@
         'OPTIONS': {
             'MAX_ENTRIES': 1000,
         },
+    },
+    'regs_gov_cache': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'KEY_PREFIX': 'regs.gov',
+        'LOCATION': 'redis://localhost:6379/0',
+        'TIMEOUT': 60*60*24,
+        'OPTIONS': {
+            'IGNORE_EXCEPTIONS': True,
+        },
```

Comment:
I was lulled into thinking that we could use `redis` as we were setting `IGNORE_EXCEPTIONS`, but it does makes sense to keep it all out of -site.